bigquery_etl/cli/backfill.py

```diff
@@ -396,9 +404,19 @@ def initiate(
         destination_table=backfill_staging_qualified_table_name,
     )
 
+    # override with billing project from backfill entry
+    if entry_to_initiate.billing_project is not None:
+        billing_project = entry_to_initiate.billing_project
+    elif not billing_project.startswith("moz-fx-data-backfill-"):
+        raise ValueError(
+            f"Invalid billing project: {billing_project}.  Please use one of the projects assigned to backfills."
+        )
+        sys.exit(1)
```

Comment:
See above comment on CLI option?
```suggestion
    else: 
        billing_project = DEFAULT_BILLING_PROJECT
```